pkg/services/notification/handler.go

```diff
@@ -24,7 +24,18 @@ import (
 func (s *Server) SetServiceConfig(ctx context.Context, req *pb.ServiceConfig) (*pb.SetServiceConfigResponse, error) {
 	err := ValidateSetServiceConfigParams(ctx, req)
 	if err != nil {
-		return nil, err
+		return nil, gerr.NewWithDetail(ctx, gerr.InvalidArgument, err, gerr.ErrorValidateEmailService)
```

Comment:
The return of func ValidateSetServiceConfigParams is already gerr, and the msg is better, why use gerr.NewWithDetail again?